lib/tasks.js

```diff
@@ -70,12 +70,10 @@ exports.parseGlobals = function() {
       return _.startsWith(value, '-');
     });
   }
-
-  // Return the globalz
-  return parse(globz, {
-    alias: {'verbose': ['v'], 'help': ['h']},
-    count: ['v']
-  });
+  else {
+    globz = [];
+    return globz;
```

Comment:
@serundeputy i dont think this line is needed?

Reply:
w/out the `return` line it doesn't work but with it it breaks at least `lando version`

Reply:
@serundeputy right, sorry, so i think you want to add the `else` to the filtering block not the return block